src/equations/serre_green_naghdi_1d.jl

```diff
@@ -383,14 +390,127 @@ function source_terms_manufactured_reflecting(q, x, t,
     return SVector(s1, s2, zero(s1))
 end
 
-# flat bathymetry with periodic boundary conditions
+"""
+    source_terms_manufactured_reflecting(q, x, t, equations::SerreGreenNaghdiEquations1D{BathymetryVariable}, mesh)
```

Comment:
```suggestion
    source_terms_manufactured_reflecting(q, x, t,
                                                                           equations::Union{SerreGreenNaghdiEquations1D{BathymetryMildSlope},
                                                                                            SerreGreenNaghdiEquations1D{BathymetryVariable}})
```
Can you also remove the `mesh` argument in docstring of the other `source_terms_manufactured_reflecting` method above (and potentially other docstrings)?

Reply:
I mean everywhere, where there is a parameter `mesh` in the docstring, but the function does not get a parameter `mesh`. The initital conditions get the `mesh`, so we should keep it there.